R/checks.R

```diff
@@ -132,3 +132,41 @@ checkOutput <- function(output, call = parent.frame()){
   }
 }
 
+#' @noRd
+checkStudyPeriod <- function(cdm, studyPeriod) {
+  if (!isTRUE(inherits(cdm, "cdm_reference"))) {
+    cli::cli_abort("cdm must be a cdm_reference object.")
+  }
+  studyPeriod <- as.character(studyPeriod)
+  assertCharacter(studyPeriod, length = 2, na = TRUE)
```

Comment:
omopgenerics::assertCharacter

Reply:
This is an internal function from this package though

Reply:
yes! There's another issue open to handle this (#99) so no need to change it now